frontend/src/pages/universalPages/ReportProblemPage.tsx

```diff
@@ -68,7 +89,15 @@ const ReportProblemPage = () => {
                         <Form.Control.Feedback type="invalid">Champ requis</Form.Control.Feedback>
                     </Form.Group>
                     <Form.Group>
-                        <Form.Label className="fw-bold h5">Description du problème</Form.Label>
+                        <Form.Label className="fw-bold h5 mt-3">Addresse email</Form.Label>
+                        <Form.Control data-testid="nomCompanyFormulaireSoumission" type="text"
```

Comment:
I'm confused why would just not put type="email" instead of having a regex and stuff. If there is a reason you want to keep it then make it that the error message goes away when it is valid

Reply:
Also close my previous points if you adressed them

Reply:
Oh also, on my other pages when I show the redirection popup there is not header and urs does so sussy

Reply:
Good otherwise

Reply:
Society

Reply:
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAHHHHHHHHHHHH

Reply:
> I'm confused why would just not put type="email" instead of having a regex and stuff. If there is a reason you want to keep it then make it that the error message goes away when it is valid

cuz email type doesnt validate it